plugins/woocommerce/client/blocks/assets/js/atomic/blocks/product-elements/image/block.json

```diff
@@ -33,6 +34,7 @@
 			"__experimentalSkipSerialization": true
 		},
 		"spacing": {
+			"blockGap": true,
```

Comment:
I assume it's leftover. Changing it does nothing.

Reply:
Sorry, merge conflicts were harder than I thought and some of these things happened during them.